### ISSUE TITLE
`deepgram.transcription.live` has a difficult to use return type

### ISSUE THREAD

**Issue by bensalilijames** [deepgram/deepgram-node-sdk#139]
## What is the current behavior?

`deepgram.transcription.live` returns a `LiveTranscription | ErrorResponse`, which makes it a bit unergonomic to use in TypeScript, because you have to explicitly cast the result to `LiveTranscription`:

```ts
const transcriber = deepgram.transcription.live({
  model: "nova",
  // other options...
}) as LiveTranscription;
```

Since it can never be `ErrorResponse` ([see implementation here, which can only return a new instance of LiveTranscription](https://github.com/deepgram/deepgram-node-sdk/blob/e425e95d8fdf22203b77566f473bb7cd0ebbe8c6/src/transcription/index.ts#L52)) would you consider reverting this change to how it was pre 2.0.0, where it just returns `LiveTranscription`?

If it's important to keep `ErrorResponse`, would you be able to suggest a way to safely use the return value, e.g. exporting `LiveTranscription` and `ErrorResponse` from this package, and thereby adding support for:

```ts
if (transcriber instanceof LiveTranscription) {
  // happy
} else if (transcriber instanceof ErrorResponse) {
  // not happy
}
```

Thank you! 😃

**Comment by lukeocodes** [comment]
This is a really good explanation and suggestion. We have a PR in-progress to revert those changes, and I am planning on a change for the v3 SDK to change it to a pattern that i've personally got a lot more enjoyment from using

```
const { ws, error } = deepgram.transcription.live({...etc});
```

and 

```
const { result, error } = deepgram.transcription.preRecorded({...etc});
```

**Comment by lukeocodes** [comment]
closed by #142
